src/cpp/src/tokenizer.cpp

```diff
@@ -353,7 +353,8 @@ class Tokenizer::TokenizerImpl {
         std::pair<std::string, std::string> replace_str_map[] = {
             {"'}", "' }"},
             {"{'", "{ '"},
-            {".strip()", ""}
+            {".strip()", ""},
+            {"is none", "is undefined"}
```

Comment:
can we add `Meta-Llama-3.1-8B-Instruct` template to https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/tokenizer_configs.py ?

Reply:
Good Idea. If it's not a full template fix, you can add only a part the template.

Reply:
Yes, sure. When the fix is ready i will add test case for that

Reply:
done